core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/ProjectOverview.java

```diff
@@ -114,16 +112,8 @@
   private boolean allPoolsConstructed;
   private boolean allRunsCompleted;
   private boolean primaryAnalysisCompleted;
-  private final Set<MisoListener> listeners = new HashSet<>();
-
-  @ManyToMany(targetEntity = UserImpl.class)
-  @JoinTable(name = "ProjectOverview_Watcher", joinColumns = { @JoinColumn(name = "overviewId") },
-      inverseJoinColumns = { @JoinColumn(name = "userId") })
-  private Set<User> watchUsers = new HashSet<>();
-
   @Transient
-  // not Hibernate-managed
-  private Group watchGroup;
+  private final Set<MisoListener> listeners = new HashSet<>();
```

Comment:
Then please delete `ProjectOverview_Watcher` table create and insert statements.